src/components/common/FilterGroup/filters/FilterMultipleSelection.tsx

```diff
@@ -33,16 +32,33 @@ export default function MultipleSelection(props: Props): JSX.Element {
     setSelections(props.values);
   }, [props.values]);
 
-  const handleChange = (value: string | null) => {
-    const updatesValues = [...selections];
+  const optionsMap = useMemo(
+    () =>
+      new Map<string, string>(
+        props.options.filter((opt) => opt.value && opt.label).map((opt) => [opt.value!, opt.label!])
```

Comment:
never used this construction before.. does the map get created from the array of tuples? first element key and second element value? just curious.

Reply:
Yep, that's how this works, `new Map<K,V>(...)` takes a list of tuples like `[[K1, V1], [K2, V2], ..., [Kn, Vn]]`